domains/certificates/Query.API/API/Query.API/ApiModels/Requests/CreateContract.cs

```diff
@@ -13,10 +14,19 @@ public class CreateContract
     /// <summary>
     /// Starting date for generation of certificates in Unix time seconds
     /// </summary>
+    [JsonPropertyName("startDate")]
```

Comment:
why specify the json property name here?

Reply:
deleted